src/components/montecarlo/histogram-panel.tsx

```diff
@@ -171,8 +152,11 @@ export class HistogramPanel extends BaseComponent<IProps, IState>{
       const numPoints = 300;
       const title = "Chart " + (this.stores.chartsStore.charts.length + 1);
       const data: number[] = [];
+      const shift = Math.random();
       for (let i = 0; i < numPoints; i++) {
-        data.push(Math.floor(this.randomG(3) * (kTephraMax + 1)));
+        let r = this.randomG(3) + shift;
+        if (r > 1) r = r - 1;
+        data.push(Math.floor(r * (kTephraMax + 1)));
```

Comment:
Simple shifting of the random histogram numbers to facilitate testing the different risk levels.  The rest of the changes in this module are just refactors.  Some code was moved to `monte-carlo.ts` for easier access.